packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -102,7 +103,7 @@ class Paginator extends React.Component {
         return;
       }
       pageButtons.push((
-        <button
+        <PaginatorButton
           aria-label={intl.formatMessage({ id: 'Terra.paginator.pageIndex' }, { pageNumber: val })}
```

Comment:
These aria attributes aren't going to get applied because you don't spread `customProps` in `PaginatorButton`. I would suggest just making defining props for them instead of using customProps anyway.

Reply:
Good catch! I added props for the various attributes: [243ad46](https://github.com/cerner/terra-core/pull/2686/commits/243ad46953a80fa9e1f6fa19c0924da3ab009538)